mmv1/third_party/terraform/tests/resource_compute_region_security_policy_rule_test.go.erb

```diff
@@ -0,0 +1,326 @@
+  region          = "asia-southeast1"
+  description     = "basic rule post update"
+  action          = "deny(403)"
+  priority        = 100
+  preview         = true
+  match {
+    versioned_expr = "SRC_IPS_V1"
+    config {
+      src_ip_ranges = ["172.16.0.0/12"]
+    }
+  }
+}
+`, context)
+}
+
+func TestAccComputeRegionSecurityPolicyRule_regionSecurityPolicyRuleNetworkMatchUpdateExample(t *testing.T) {
+	context := map[string]interface{}{
+		"random_suffix": RandString(t, 10),
+	}
+
+	VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckComputeRegionSecurityPolicyRuleDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeRegionSecurityPolicyRule_regionSecurityPolicyRuleNetworkMatchBasicExample(context),
+			},
+      {
+        ResourceName:      "google_compute_region_security_policy_rule.policy_rule_network_match",
+        ImportState:       true,
+        ImportStateVerify: true,
+      },
+      {
+        Config: testAccComputeRegionSecurityPolicyRule_regionSecurityPolicyRuleNetworkMatchUpdateExample(context),
+      },
+      {
+        ResourceName:      "google_compute_region_security_policy_rule.policy_rule_network_match",
+        ImportState:       true,
+        ImportStateVerify: true,
+      },
+      {
+        Config: testAccComputeRegionSecurityPolicyRule_regionSecurityPolicyRuleNetworkMatchUpdate2Example(context),
+      },
+      {
+        ResourceName:      "google_compute_region_security_policy_rule.policy_rule_network_match",
+        ImportState:       true,
+        ImportStateVerify: true,
+      },
+      {
+				Config: testAccComputeRegionSecurityPolicyRule_regionSecurityPolicyRuleNetworkMatchBasicExample(context),
+			},
+      {
+        ResourceName:      "google_compute_region_security_policy_rule.policy_rule_network_match",
+        ImportState:       true,
+        ImportStateVerify: true,
+      },
+		},
+	})
+}
+
+func testAccComputeRegionSecurityPolicyRule_regionSecurityPolicyRuleNetworkMatchBasicExample(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_compute_region_security_policy" "policyddosprotection" {
+  region      = "asia-southeast1"
+  name        = "tf-test-policyddos%{random_suffix}"
+  description = "region security policy for network match"
+  type        = "CLOUD_ARMOR_NETWORK"
+  ddos_protection_config {
+    ddos_protection = "ADVANCED_PREVIEW"
+  }
+}
+  
+resource "google_compute_network_edge_security_service" "edge_sec_service" {
+  region          = "asia-southeast1"
+  name            = "tf-test-edgesec%{random_suffix}"
+  security_policy = google_compute_region_security_policy.policyddosprotection.self_link
+}
+  
+resource "google_compute_region_security_policy" "policynetworkmatch" {
+  region      = "asia-southeast1"
+  project     = "ciandt-cloudarmor-gcam"
```

Comment:
Is this supposed to be a different project than the rest of the configuration? (same with others below)

Reply:
No. Thanks for catching this.
Done.

Reply:
These didn't get updated in your last commit, just the test names!

Reply:
Done